app/controllers/trainees/award_recommendations_controller.rb

```diff
@@ -6,8 +6,14 @@ def create
       if OutcomeDateForm.new(trainee).save! && trainee.submission_ready?
         trainee.recommend_for_award!
 
-        Dttp::RecommendForAwardJob.perform_later(trainee)
-        Dttp::RetrieveAwardJob.perform_with_default_delay(trainee)
+        if FeatureService.enabled?(:persist_to_dttp)
```

Comment:
I've feature flagged this in the same way as we flagged the TRN submission flow.